modules/dists/SparseBlockDist.chpl

```diff
@@ -229,7 +229,13 @@ class SparseBlockDom: BaseSparseDomImpl {
       yield i;
   }
 
-  iter these(param tag: iterKind) where tag == iterKind.standalone {
+  iter these(param tag: iterKind) where tag == iterKind.standalone &&
+    // Ensure it is legal to invoke the standalone iterator
+    // on locDom.mySparseBlock below.
+    __primitive("method call resolves",
+                locDoms[createTuple(rank,int,0)].mySparseBlock._value,
+                "these", tag)
+  {
```

Comment:
I understand why this logic might be worthwhile for array views but I'm scratching my head a bit about SparseBlock. Shouldn't all of the ._values in SparseBlock just always have serial/standalone/leader/follower iteratiors? Is there some reason that is not a good idea? Aren't they all controlled by our implementation of SparseBlock?

In other words, what is it about SparseBlock that required this, separately from Block or Cyclic or ... ?